src/components/breadcrumb/Breadcrumb.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import * as React from 'react';
 import { injectIntl } from 'react-intl';
```

Comment:
I think this can be simplified: `import { injectIntl, type InjectIntlProvidedProps } from 'react-intl';`